D1673/P1673.md

```diff
@@ -6342,10 +6340,9 @@ _`transpose-extents-t`_`<InputExtents>` names the `extents` type
       `OutputExtents::static_extent(r)`
       for 0 ≤ `r` < `InputExtents::rank()-2`.
 
-[3]{.pnum} *Preconditions:* `InputExtents`
-is a specialization of `extents`.
+[3]{.pnum} *Constraints:* `InputExtents::rank()` equals 2.
```

Comment:
It feels like the opposite would be easier: Mandate that the rank equals 2, and Constrain that `InputExtents` is a specialization of `extents`.